Dockerfile

```diff
@@ -1,34 +1,11 @@
-# Build the manager binary
-FROM golang:1.19 as builder
-ARG TARGETOS
-ARG TARGETARCH
+FROM gcr.io/distroless/static:nonroot
```

Comment:
Can we add a comment to this file that says it's intended to be used with `make docker-build` or `make release`?